media/css/base/mozid-modal.less

```diff
@@ -0,0 +1,158 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+html.noscroll body {
+    overflow: hidden;
+}
+
+#modal {
+    background: #000;
+    background: rgba(0, 0, 0, 0.85);
+    width: 100%;
+    height: 101%;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 9999999;
+    overflow: auto;
+
+    .window {
+        padding: 20px;
+        color: #333;
+
+        .inner {
+            position: relative;
+
+            header {
+                display: block;
+                height: auto;
+                .font-size(60px);
+                text-align: center;
+                color: #fff;
+                .open-sans-light;
+                .transition;
+            }
+
+            #modal-close {
+                z-index: 99;
+                position: absolute;
+                top: -20px;
+                right: 9px;
+                cursor: pointer;
+
+                a {
+                    // TODO: remove <a> element from mozilla-modal.js with mozID rollout
+                    display: none;
+                }
+
+                .button {
+                    background-color: transparent;
+                    // png fallback for IE8
+                    background-image: url("data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAQAAABKfvVzAAAAhElEQVR4AY3Rsa0DMRDE0N0CrtAPh1Yv1kHF8kebDQFhUr5oij9+NHWxZvEUG9gXpDnAt2he4KUv8kMXTkJOFU5CPiCQlA8IRPIBgUg+IJCQDxAS8wGBhHyAkJgbOGC/WH7sF8vbrrRcr/RciOdCPBfiuRDPhXgupFiSZ7KLhxXyTDaff714nz5yfKKNAAAAAElFTkSuQmCC");
```

Comment:
I know :(

If this was just one page I wouldn't even have brought it up. But seeing as this could appear on any page I thought we should probably consider it still here.